ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonDemoController.swift

```diff
@@ -151,6 +151,8 @@ extension ButtonStyle {
             return "Floating accent"
         case .floatingSubtle:
             return "Floating subtle"
+        case .outline:
+            return "Outline accent"
```

Comment:
should this just be `return "Outline"`?

Reply:
I kept it as `"Outline accent"` because that's what the `.outline` case said before, but it shouldn't be showing either way right now. And they styling is still the same as `.outlineAccent`.